distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -819,6 +819,77 @@ type Table
             on_problems.attach_problems_before problems <|
                 self.updated_context_and_columns new_ctx new_columns
 
+    ## Returns a new table with a chosen subset of columns left unchanged and
+       the other columns pivoted to rows with a single name field and a single
+       value field.
+
+       Arguments:
+       - id_fields: Set of fields to remain as columns. These values will be
+         repeated for each data field that is pivoted.
+       - name_field: The name of the field that will contain the names of the
+         pivoted fields. If this name is already in use, it will be renamed
+         with a numeric suffix.
+       - value_field: The name of the field that will contain the values of the
+         pivoted fields. If this name is already in use, it will be renamed
+         with a numeric suffix.
+       - on_problems: Specifies how to handle problems if they occur, reporting
+         them as warnings by default.
+
+         The following problems can occur:
+         - If a column in columns is not in the input table, a
+           `Missing_Input_Columns`.
+         - If duplicate columns, names or indices are provided, a
+           `Duplicate_Column_Selectors`.
+         - If a column index is out of range, a `Column_Indexes_Out_Of_Range`.
+         - If two distinct indices refer to the same column, an
+           `Input_Indices_Already_Matched`, with the column included the first
+           time it is matched.
+         - If there are no columns to transpose, a `No_Input_Columns`.
+    transpose : Vector Text | Column_Selector -> Text -> Text -> Problem_Behavior -> Table
+    transpose self (id_fields = Column_Selector.By_Name []) (name_field="Name") (value_field="Value") (on_problems = Report_Warning) =
+        ## Avoid unused arguments warning. We cannot rename arguments to `_`,
+           because we need to keep the API consistent with the in-memory table.
+        _ = [id_fields, name_field, value_field, on_problems]
```

Comment:
Btw. this makes me think if we should consider some proper solution for it like an `@unused` annotation?